policyengine_canada/variables/gov/provinces/ab/benefits/marital_status/ab_marital_status_credit.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_canada.model_api import *
+
+
+class ab_marital_status_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Alberta marital status credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.AB
+
+    def formula(household, period, parameters):
+        spouse_income = add(household, period, ["separated_spouse_income"])
```

Comment:
I think it's the opposite: you can take a credit for your spouse's income only if not separated